CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Bug fixes
 
+- Attributes of `pybamm.Simulation` objects (models, parameter values, geometries, choice of solver, and output variables) are now private and as such cannot be edited in-place after the simulation has been created ([#3267](https://github.com/pybamm-team/PyBaMM/pull/3267)
```

Comment:
Can you sort the entries in this section so they are in decreasing PR number?

Reply:
Done